### PR TITLE
Added Policy for releasing Information [Information Release Policy.md]

### PR DESCRIPTION
**Pull request by TamaraDale**
New Policy around releasing information.

### CODE REVIEW

File: policies/Information Release Policy.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  information.
+* whether the information is suitable and appropriate to be released:
+  * fit for purpose. Fit for purpose refers to the closeness of correspondence 
+    between the characteristics of the information provided and its intended 
+    purpose. Poor fit means that the information is unlikely to meet the needs 
+    of those requesting the information.
+  * is a suitable quality for use;
+  * is accurate and complete;
+  * is reliable or whether it is subject to further change (outside of regular 
+reviews).
```
Comment (levitte) [inline policies/Information Release Policy.md:42]: Nit: indent with 4 spaces

File: policies/information-release-policy.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    purpose. Poor fit means that the information is unlikely to meet the needs 
+    of those requesting the information.
+  * is a suitable quality for use;
+  * is accurate and complete;
+  * is reliable or whether it is subject to further change (outside of regular 
+    reviews).
+* resource availability. Information release requests can involve a 
+  significant commitment of resources, and sometimes specialised/technical 
+  resources.
+* whether OPENSSL can ensure it is used only for the purpose for which it was
```
Comment (t-j-h) [inline policies/information-release-policy.md:46]: Not OPENSSL but OpenSSL

Comment (TamaraDale) [reply policies/information-release-policy.md]: All fixed :-)

**Review by mattcaswell — APPROVED** [review]

**Comment by paulidale** [comment]
Vote: +1

**Comment by mattcaswell** [comment]
Vote: +1

File: policies/information-release-policy.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+    reviews).
+* resource availability. Information release requests can involve a 
+  significant commitment of resources, and sometimes specialised/technical 
+  resources.
+* whether OpenSSL can ensure it is used only for the purpose for which it was 
+  released and that it will not be disclosed  to other parties unless previously 
+  agreed to or provided for by law.
+
+Where OpenSSL may be required to release information based on legislative 
+requirements, subpoenas or other legal discovery obligations. OpenSSL will
```
[the remark this answers is not in the document]
Comment (paulidale) [reply policies/information-release-policy.md:51]: IMO: this should be fixed after the vote concludes as a typographic change (which has a lower acceptance bar).

**Comment by iamamoose** [comment]
vote: 0

**Comment by levitte** [comment]
Vote: +1

**Comment by t-j-h** [comment]
Vote +1

**Comment by kroeckx** [comment]
Voting 0

File: policies/information-release-policy.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (paulidale) [reply policies/information-release-policy.md:51]: @TamaraDale, could you create a PR with this change please?

File: votes/vote-20220928-releasing-information-policy.txt [hunk 4]
```diff
@@ -0,0 +1,13 @@
+Topic: Accept the policy for releasing information as at c78f885
+Proposed by: Pauli
+Issue link: https://github.com/openssl/general-policies/pull/24
+Opened: 2022-09-28
+Closed: 2022-10-05
+Accepted:  yes  (for: , against: 0, abstained: 0, not voted: )
```
Comment (levitte) [inline votes/vote-20220928-releasing-information-policy.txt:6]: Why do we have this line if we don't bother updating it properly?

**Pull request closed without merging**